TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -61,7 +61,7 @@ class RouteOptionsViewController: UIViewController, UITableViewDelegate, UITable
     let reachability: Reachability? = Reachability(hostname: Network.ipAddress)
 
     var banner: StatusBarNotificationBanner = {
-        let banner = StatusBarNotificationBanner(title: "No internet connection. Retrying...", style: .danger)
+        let banner = StatusBarNotificationBanner(title: "No internet connection", style: .danger)
```

Comment:
Mihir said the "retrying..." added a live factor to the banner. Do you not like it?

Reply:
I tested it. It doesn’t re-try if you reconnect Internet

Reply:
Something to work on post release. I've started adding issues for stuff like that, feel free to add